corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -86,20 +86,23 @@ def process_bulk_app_translation_upload(app, workbook, expected_headers, lang=No
 
         if is_single_sheet(sheet.worksheet.title):
             module_or_form = None
+            module_or_form_rows = {}
```

Comment:
Curious what the reason for this change is. Ordering of error messages?

Reply:
Not all rows have `unique_id` populated. I'm processing the rows that have a `unique_id` first, to build a dictionary that maps `sheet_name` to `unique_id`. That way I can look up subsequent forms and modules using their `unique_id` when the row only gives their `sheet_name`.

Reply:
aha, got it